### DISCUSSION TITLE
android apk

### DISCUSSION THREAD

**Discussion by senolfndk** [FreeTubeApp/FreeTube#2465] [General]
gives an installation error
![Screenshot_20220810-142244_Package installer](https://user-images.githubusercontent.com/52818836/183889575-30c5e4b5-9af5-4024-ad2c-bfc54f9407b6.jpg)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
1. Where did u get the install file?
2. What is the name of the install file?
3. On what deceive u tried to install it?

**Reply by senolfndk** [reply]
https://github.com/FreeTubeApp/FreeTube/suites/7751383887/artifacts/325512069
freetube_0.17.0 nightly 1881_alpine_arm64.apk

![Screenshot_20220810-204838_Settings](https://user-images.githubusercontent.com/52818836/183981829-493114f6-d32c-4e16-a6ea-88f14293444e.jpg)
![Screenshot_20220810-204842_Settings](https://user-images.githubusercontent.com/52818836/183981843-a14bf183-6377-49db-986e-0f7fbd6f68f6.jpg)

**Comment by PrestonN** [comment]
This file is not compatible with Android, but is actual a binary for Alpine Linux (hence why `alpine` is in the filename). Alpine Linux binaries also use the `.apk` extension.

**Comment by ChunkyProgrammer** [comment]
As mentioned by Preston, that is an Alpine Linux file. There is a fork of FreeTube by @MarmadileManteater that works on Android that you could try out if you'd like https://github.com/MarmadileManteater/FreeTubeCordova/releases
